blockchain/fullblocks_test.go

```diff
@@ -124,183 +129,182 @@ func chainSetup(dbName string, params *chaincfg.Params) (*blockchain.BlockChain,
 	return chain, teardown, nil
 }
 
-// TODO(Litecoin): disable test until testfile is available
 // TestFullBlocks ensures all tests generated by the fullblocktests package
 // have the expected result when processed via ProcessBlock.
-// func TestFullBlocks(t *testing.T) {
```

Comment:
no missing testfiles. need to investigate. looks like this might be using a custom signet, which hasn't been properly implemented, causing failures. not sure.